apps/web/src/pages/templates/workflow/digest/ScheduleMonthlyFields.tsx

```diff
@@ -3,7 +3,7 @@ import { Controller, useFormContext } from 'react-hook-form';
 import { DigestUnitEnum, OrdinalEnum, OrdinalValueEnum, MonthlyTypeEnum, DigestTypeEnum } from '@novu/shared';
 
 import { When } from '../../../../components/utils/When';
-import { colors, Select } from '../../../../design-system';
+import { colors, Select } from '@novu/design-system';
```

Comment:
When is now in the Design System.

Reply:
Yes it is used by some components, maybe we should not export it?

Reply:
What I meant is that you can import it from the package so we would not need line 5.